### ISSUE TITLE
Incompatability with Clojure 1.9-alpha17

### ISSUE THREAD

**Issue by aredington** [JonyEpsilon/lein-gorilla#10]
Hi!

I dig this work, thanks a lot for all the time and energy you've invested in Gorilla so far.

I was trying to start up a notebook to share some thoughts with my team using lein-gorilla, and ran into problems with getting it to work. Namely, somewhere in the dependency chain lein-gorilla ends up pulling in `[ring/ring-json "0.3.1"]` which ultimately ends up loading `[ring/ring-core "1.2.2"]` which has an ns decl that uses `(require ...)` instead of `(:require ...)`.

# Steps to reproduce:

Create a new lein project. In deps include `[org.clojure/clojure "1.9.0-alpha17"]`. In plugins include `[lein-gorilla "0.4.0"]`. Run `lein gorilla`.

# Symptoms

Clojure spec will complain about the ns decl in ring.util.mime-types.

# Workarounds

I found I could work around this with the following dependencies decl:

```[[org.clojure/clojure "1.9.0-alpha17"]
                 [gorilla-repl "0.4.0" :exclusions [ring/ring-json compojure]]
                 [ring/ring-json "0.4.0"]
                 [compojure "1.6.0"]]
```

and the following alias:

```
:aliases {"gorilla" ["run" "-m" "gorilla-repl.core/run-gorilla-server" "{:port \"3000\" :ip \"127.0.0.1\" :nrepl-port 0 :gorilla-version \"0.4.0\" :project \"<my project here>\", :gorilla-options nil}"]}
```

Trying to add exclusions to :plugins didn't work for me.

**Comment by tmountain** [comment]
I'm also a big fan of this project, and I use gorilla daily in my professional work. Unfortunately, I'm having the same issue as aredington, and I hate to have to resort to workarounds whenever I use gorilla with a modern clojure build.

**Comment by behrica** [comment]
I had an other problem with gorilla and "recent" libraries. see here
https://github.com/sbelak/huri/issues/7

I got it finaly working by applying a pending CR to "gorilla-repl":
https://github.com/JonyEpsilon/gorilla-repl/pull/275

and the updating lein-gorilla with the new version.

**Comment by Engelberg** [comment]
The lein-gorilla plugin seems to include an old version of nrepl that isn't compatible with Clojure 1.9.  Adding the lein-gorilla plugin broke my ability to launch a Cider repl and it took me hours to track the problem down to lein-gorilla.

**Comment by benfb** [comment]
Hey everyone, thanks for your work on this! I just pushed the latest release of gorilla and the Lein plugin to a different repository/fork (`org.clojars.benfb/lein-gorilla`). The code for that plugin is located here: https://github.com/benfb/lein-gorilla. Jony has been mostly inactive, but I didn't want to make code changes without his approval under his name, so I created the other repository. Please raise any issues you have there. Thanks!

The newly-released version (0.5.0) of the forked plugin should fix this. To use it, just change the `lein-gorilla "0.4.0"` plugin to `org.clojars.benfb/lein-gorilla "0.5.0"`.

**Comment by blevin** [comment]
Thanks benfb!  For anyone following this path, I hit another error with 0.5.0 (see below) which was fixed by going to the current latest, 0.6.0:
```
clojure.lang.Compiler$CompilerException: Syntax error compiling var at (gorilla_repl/render_values_mw.clj:40:1).
#:clojure.error{:phase :compile-syntax-check, :line 40, :column 1, :source "gorilla_repl/render_values_mw.clj", :symbol var}
 at clojure.lang.Compiler.analyzeSeq (Compiler.java:7114)
```
